crates/ethereum/node/src/node.rs

```diff
@@ -362,6 +362,7 @@ where
             .with_head_timestamp(ctx.head().timestamp)
             .kzg_settings(ctx.kzg_settings()?)
             .with_local_transactions_config(pool_config.local_transactions_config.clone())
+            .set_tx_fee_cap(u128::try_from(ctx.config().rpc.rpc_tx_fee_cap)?)
```

Comment:
we can use `ctx.config().rpc.rpc_tx_fee_cap.saturating_to` here but I think we should do this on CLI level when we parse the value